docs/content/sre/tenant-operator/customresources.md

```diff
@@ -245,6 +258,10 @@ Also you can define custom variables in `Template` and `TemplateInstance` . The
   <summary> Helm Chart Templates</summary>
   <p>Instead of manifests, a Template can specify a Helm chart that will be installed (using Helm template) when the Template is being instantiated.</p>
 </details>
+<details open>
+  <summary>Resource Mapping Templates</summary>
+  <p>The templates can be specified with secrets and configMaps that can be mapped from one tenant's namespace to some other tenant's namespace or within a tenant's namespace.</p>
```

Comment:
A template can be used to map secrets and configMaps from one tenant's namespace to another tenant's namespace, or within a tenant's namespace.

Reply:
The rest of the points are using a more active voice, so this point should too. It also sounds clearer :)